Gemfile

```diff
@@ -44,30 +45,28 @@ gem 'fog'
 gem 'thin'
 gem 'roadie-rails', '~> 1.1.0'
 gem 'sequenced', '~> 2.0.0'
-gem 'closure_tree', '~> 5.2.0'
+# gem 'closure_tree', '~> 5.2.0'
 gem 'bing_translator'
 gem 'http_accept_language'
 gem 'intercom'
-gem 'intercom-rails'
-gem 'underscore-rails'
-gem "font-awesome-sass",  '~> 4.3.2'
+# gem 'intercom-rails'
+# gem 'underscore-rails'
+# gem "font-awesome-sass",  '~> 4.3.2'
 gem 'coffee-rails'
 gem 'sprockets-rails', require: 'sprockets/railtie'
 gem 'uglifier'
 gem 'bootstrap-sass', '~> 3.3.4'
 gem 'sass-rails', '>= 3.2'
-gem 'autoprefixer-rails'
-gem 'modernizr-rails'
-gem 'jquery-fileupload-rails'
-gem 'momentjs-rails'
+# gem 'autoprefixer-rails'
+# gem 'modernizr-rails'
+# gem 'jquery-fileupload-rails'
+# gem 'momentjs-rails'
 gem 'ruby-progressbar'
-gem 'responders', '~> 2.0'
+# gem 'responders', '~> 2.0'
 gem 'activerecord_any_of'
 gem 'rack-attack', '~> 4.2.0'
-gem 'ahoy_matey'
+gem 'ahoy_matey', '~> 1.1.1'
 gem 'ahoy_email'
-gem 'oj'
```

Comment:
oj is supposedly a faster json implementation. I'm happy to take it out but it's not beta